hack/k8s/example-coord.template.yaml

```diff
@@ -69,8 +72,14 @@ spec:
         - mountPath: /tmp/gac
           name: google-application-credentials
         env:
+
         - name: GOOGLE_APPLICATION_CREDENTIALS
           value: /tmp/gac/google-application-credentials.json
+
+        - name: dm_fuse_params
+          value: /fuse-params.yaml
```

Comment:
Where does this file come from in this example? Is it built into the datamon sidecar container?

Reply:
it's built via `hack/fuse-demo/write_fuse_params.go` ...

... external to docker then there's an `ADD` in the dockerfile.

let's sync up re. pipelines tool to find out how to tighten up integration.